### DISCUSSION TITLE
Unable to make pull request: "Validation failed: must be a collaborator"

### DISCUSSION THREAD

**Discussion by niparx** [cross-language-cpp/djinni-generator#154] [Q&A]
We have implemented a small feature and would like to share it with community. But, we can't make pull request: `Validation failed: must be a collaborator`
So, question is: how to become a collaborator? May be I have missed, but I did not find anything about that... :/

What's PR all about?
It's about this issue: https://developer.apple.com/documentation/swift/renaming-objective-c-apis-for-swift
If I'm using generated code in Swift project, then Swift will rename my methods into something else (they have some naming rules). That, basically brakes all thing: API then looking totally different. There is a macro `NS_SWIFT_NAME`, where you can put a name, which Swift will use (something like "I know what I'm doing. Please, Swift, leave method name as it is written in thos macro."). So, there is an option for that: to add or not to add this macro to every method of API.

**Comment by a4z** [comment]
Thank you for trying to contribute to this project! 
This is unfortunate, it should not be like this. We had some contributions in the past, never a problem.

This seems to be a new issue for some users on GitHub, not just our repo
see https://github.com/orgs/community/discussions/61472

It might be some side effect of GitHub fighting spam coming in via pull requests, but I don't know

Can you point me to your repository to see what I can do?

**Reply by niparx** [reply]
@a4z yap, sure: https://github.com/FATMAP/djinni-generator/tree/FM-2707_Prevent_Renaming_of_SDK_methods_in_swift

**Reply by a4z** [reply]
I can't find a reason. There is a new moderation feature that allows setting limits temporarily, but I do not think anyone ever used that. (at least I hope so)

I can invite you to the organization, which should in any case fix it.
But whom shall I invite? you, or the author of the change, or you both?

**Reply by niparx** [reply]
it's ok to invite both of us.

**Reply by a4z** [reply]
OK, just sent both invitations to the cross-language-cpp organization, hope that will solve the problem
